jinad/api/endpoints/flow.py

```diff
@@ -120,16 +171,16 @@ async def _fetch(
     """
     try:
         # TODO: Fix - This fails with the inherited class
-        # yaml_spec = flow_wrapper.f.yaml_spec
-        # if yaml_only:
-        #     return Response(content=yaml_spec,
-        #                     media_type='application/yaml')
         with flow_store._session():
-            host, port_expose = flow_store._get(flow_id=flow_id)
+            host, port_expose, yaml_spec = flow_store._get(flow_id=flow_id)
+
+        if yaml_only:
```

Comment:
what does tis parameter do?

Reply:
This decides whether to send just `yaml_spec` for a particular flow. This will help us just an `application/yaml` type directly, which can help the dashboard

Without `yaml_only`
![image](https://user-images.githubusercontent.com/9050737/97336574-2edf7900-18a5-11eb-9f42-864a3d89820c.png)

With `yaml_only`
![image](https://user-images.githubusercontent.com/9050737/97336686-49195700-18a5-11eb-9469-54db1051fe73.png)